frontend/routes/auth/settings/telemetry-section.spec.tsx

```diff
@@ -0,0 +1,112 @@
+import { render, screen, fireEvent } from '@testing-library/react'
```

Comment:
Could we add a test checking that when we toggle the yes/no buttons that they do the following:
1. change to the correct state
2. Only one radio button is ever toggled at one time

Reply:
I also think the correct hierachy of testing here is this:

unit test:
1. Check we can toggle yes, 
2. Check we can toggle no
3. Check only one radio button selected at either time
4. Check there is always a radio button selected by default(?), as long as the default is consistent

e2e test:

1. Update the configuration in settings
2. refresh
3. check change is retained

Reply:
Added unit tests, if you don't mind adding the e2e test 👍